Engines/Wine/Verbs/VK9/script.js

```diff
@@ -102,14 +103,11 @@ class VK9 {
         const wine = new Wine();
         const wizard = SetupWizard(InstallationType.VERBS, "VK9", Optional.empty());
 
-        wine.prefix(container);
```

Comment:
Are you sure this can be deleted?

Reply:
Weird. I swear I did not remove that line.